### ISSUE TITLE
Clarification needed for the acronym "URI" in the context of RRID

### ISSUE THREAD

**Issue by zoe-translates** [common-workflow-language/user_guide#409]
https://github.com/common-workflow-language/user_guide/blob/b6c2f66a69f2eb63ba37b9a012eaa73743320701/src/topics/specifying-software-requirements.md?plain=1#L29

Should this be _uniform_ resource identifier? (Although I don't think the RRID has a formal URL scheme or URN namespace yet.)

**Comment by mr-c** [comment]
@zoe-translates That's correct. Really it needs to be any persistent identifier ([PID](https://en.wikipedia.org/wiki/Persistent_identifier)) in the form of an URI/IRI, so we can take the RRID [CURIE](https://en.wikipedia.org/wiki/CURIE) and express it as a full URI/IRI using a resolution service like identifiers.org.

So that's the full technical explanation; but that is overly complex for this section, I think.

I would happily review and clarification or simplification you would suggest!

**Comment by mr-c** [comment]
> Based on my understanding of the [`SoftwarePackage`](https://www.commonwl.org/v1.2/Workflow.html#SoftwarePackage) in the specification, the `specs` field should only list full IRIs, rather than compact IDs -- is that correct?

That is correct; full URI/IRIs only.

> Am I correct in assuming that the correct "resolution" of a compact ID to IRI (here using the identifiers.org redirect) is something the user should do, in order to maximize the interoperability of the CWL file being written?

Yes
